ngraph_bridge/ie_executable.h

```diff
@@ -40,11 +40,18 @@ class IE_Executable final : public Executable {
             const vector<shared_ptr<ngraph::runtime::Tensor>>& inputs) final;
 
  private:
+  bool call_trivial(const vector<shared_ptr<ngraph::runtime::Tensor>>& outputs,
+                    const vector<shared_ptr<ngraph::runtime::Tensor>>& inputs);
+
   InferenceEngine::CNNNetwork m_network;
   InferenceEngine::InferRequest m_infer_req;
+  string m_device;
   // This holds the parameters we insert for functions with no input parameters
   vector<pair<string, shared_ptr<ngraph::runtime::Tensor>>> m_hoisted_params;
-  string m_device;
+  // This keeps track of whether the originial function was either
+  // constant-folded
+  // or was simply a trivial identity function
```

Comment:
I am a little confused with the comment - won't constant folding lead to trivial functions?

Reply:
Yes, but not vice versa. Identity functions won't be constant-folded.

Reply:
So this bool is for keeping track of the trivial identity functions?

Reply:
It keeps track of both.